docs/api.rst

```diff
@@ -6019,10 +6019,6 @@ ActionRow
     :members:
     :exclude-members: add_select
 
-    .. method:: add_select
```

Comment:
this documentation was fine to stay, I think my comment didn't make sense. 

This should stay documented.

Reply:
The change I made was independent of your comment, I removed `add_select` again as the other "old" names are also not documented, which is intentional.